flo-runner/src/main/java/com/spotify/flo/context/ForkingExecutor.java

```diff
@@ -129,7 +130,7 @@ void start() {
       }
       log.debug("serializing closure");
       try {
-        PersistingContext.serialize(f, closureFile);
+        Serialization.serialize(f, closureFile);
       } catch (Exception e) {
```

Comment:
Is it intended to catch all kinds of exceptions here? And similar code below.

Reply:
Oops